### ISSUE TITLE
CPU governor: powersave when on battery and performance when plugged in

### ISSUE THREAD

**Issue by bonsairobo** [bonsairobo/MyNixOs#14]
Currently have this in hardware-configuration.nix:

```
powerManagement.cpuFreqGovernor = lib.mkDefault "powersave";
```

Not sure how to configure this behavior that depends on whether the laptop is plugged in.

**Comment by bonsairobo** [comment]
It's possible to dynamically change the governor policy with this command:

```
sudo cpupower frequency-set --governor performance
```

**Comment by bonsairobo** [comment]
Finally found this: https://discourse.nixos.org/t/how-to-switch-cpu-governor-on-battery-power/8446/4

**Comment by bonsairobo** [comment]
Resolved by 84a7ea1
